Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml

```diff
@@ -53,6 +53,12 @@ configuration:
   section: Collect
   advanced: true
   required: false
+- display: Disable reputation lookups for private IP addresses
```

Comment:
hi @omerKarkKatz , this parameter, as in the integration for VirusTotal, is needed in the AbuseIPDB to avoid consuming API calls on private IP addresses when XSOAR invokes the "!ip" command to perform indicator enrichment or in other use cases.